src/graph/commands/epicurve/epicurvesource.pas

```diff
@@ -0,0 +1,134 @@
+unit epicurvesource;
```

Comment:
This unit does not seem to be used anywhere in the code? IS this a mistake or has it been replaced by something else?
If it is not needed anymore please delete the unit.

Reply:
removed it